packages/core/src/components/DataTable/index.tsx

```diff
@@ -413,6 +413,7 @@ export default withStyles(
   theme => ({
     table_container: {
       overflowX: 'auto',
+      height: '100%',
```

Comment:
is this PR still necessary, given this change (adding the height: 100%)? https://github.com/airbnb/lunar/pull/168

Reply:
Just talked to Conglei again, I don't think #168 is the right fix since we're still struggling to reproduce the issue, so I might just abandon that PR